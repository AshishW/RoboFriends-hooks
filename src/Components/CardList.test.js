import {shallow} from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('expects to render the card component', ()=>{
    const mockRobots = [
        {
            id: 1,
            name: 'test',
            email: 'test@gmail.com'
        }
    ];
    expect(shallow(<CardList robots = { mockRobots }/>)).toMatchSnapshot();
})

//the CardList Component filters robots. So it needs robots as props, 
//that's why we'll have to create a mockRobots array 
//that contains the information of the robot inside objects.
 
