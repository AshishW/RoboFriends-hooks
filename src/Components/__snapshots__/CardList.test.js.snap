// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render the card component 1`] = `ShallowWrapper {}`;
